snuba/web/rpc/v1/endpoint_trace_item_attribute_names.py

```diff
@@ -300,7 +338,18 @@ def t(row: Row) -> TraceItemAttributeNamesResponse.Attribute:
             name=attr_name, type=getattr(AttributeKey.Type, attr_type)
         )
 
-    return list(map(t, query_res.result.get("data", [])))
+    data = query_res.result.get("data", [])
```

Comment:
nit: If `"data"` is not present in `query_res.result`, then we get an empty list, which is further extended but not persisted back to the `result`. Should it be persisted?